contracts/LockedAccount.sol

```diff
@@ -153,6 +166,7 @@ contract LockedAccount is AccessControlled, AccessRoles, TimeSource, ReturnsErro
     // this allows to unlock and allow neumarks to be burned in one transaction
     function receiveApproval(address from, uint256 _amount, address _token, bytes _data)
         onlyStates(LockState.AcceptingUnlocks, LockState.ReleaseAll)
+        acceptAgreement(from)
```

Comment:
Each investor accepts agreement by calling lock(). We can safely remove it here

Reply:
ok.